<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighlightWidget</class>
 <widget class="QWidget" name="HighlightWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="6">
    <widget class="QLineEdit" name="conditionLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="bgColorLabel">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>bgColorWidget</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ColorWidget" name="bgColorWidget"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="fgColorLabel">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="buddy">
      <cstring>fgColorWidget</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="ColorWidget" name="fgColorWidget"/>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Scope:</string>
     </property>
     <property name="buddy">
      <cstring>scopeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QComboBox" name="scopeComboBox">
     <item>
      <property name="text">
       <string>Cell</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Row</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Condition:</string>
     </property>
     <property name="buddy">
      <cstring>conditionLineEdit</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorWidget</class>
   <extends>QPushButton</extends>
   <header>ColorWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>conditionLineEdit</tabstop>
  <tabstop>bgColorWidget</tabstop>
  <tabstop>fgColorWidget</tabstop>
  <tabstop>scopeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
